vmdb/app/models/ems_refresh/save_inventory_helper.rb

```diff
@@ -1,8 +1,20 @@
 module EmsRefresh::SaveInventoryHelper
   def save_inventory_multi(type, klass, parent, hashes, deletes, find_key, child_keys = [], extra_keys = [])
+    return if hashes.nil?
```

Comment:
Is this needed?  All of the callers should really be checking this (unless maybe we want to consolidate this check as well, but maybe not for this PR).